[DOC] 77 - Update README.rst (#131)
ansys/pymaterials-manager: README.rst

Distribution
------------

If you want to create either source or wheel files, start by installing poetry and then 

> executing the build module. To install poetry we suggest using the pipx installation

Reviewer: ```suggestion
executing the build module. To install poetry we suggest using the ``pipx`` installation
```
I looked at the referenced documentation, and they tagged ``pipx`` as a literal. Whether to tag these names is random these days. We tag ``pip``, ``pipx", and ``tox`` but not ``poetry`` and other package names that are familiar words. I hate that we are are all over the place with this.

> executing the build module. To install poetry we suggest using the pipx installation

Reviewer: ```suggestion
executing the build module. To install poetry, you should use the pipx installation
```
Google developer documentation style guide (and most tech writing style guides) say avoid ``we``.
